### PR TITLE
luci-app-rosy-file-server: Add Spanish translation

### PR DESCRIPTION
**Pull request by castillofrancodamian**
Add Spanish translation.

Signed-off-by: Franco Castillo <castillofrancodamian@gmail.com>

### CODE REVIEW

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.2.3\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: es\n"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:50
+msgid "<a href='%s%s'>%s</a>"
+msgstr "<a href='%s%s'>%s</a>"
```
Comment (feckert) [inline applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: Do we need this to translate?

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:62
+msgid "<a href='%s?server_root=%s%s'>%s</a>"
+msgstr "<a href='%s?server_root=%s%s'>%s</a>"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:59
+msgid "<a href='%s?server_root=%s'>%s</a>"
+msgstr "<a href='%s?server_root=%s'>%s</a>"
```
Comment (feckert) [inline applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: Do we need this to translate?

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: es\n"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:50
+msgid "<a href='%s%s'>%s</a>"
+msgstr "<a href='%s%s'>%s</a>"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:62
+msgid "<a href='%s?server_root=%s%s'>%s</a>"
+msgstr "<a href='%s?server_root=%s%s'>%s</a>"
```
Comment (feckert) [inline applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: Do we need this to translate?

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po (hunk 1 above)
Comment (castillofrancodamian) [reply applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: To see that it was 100% completed.

Comment (castillofrancodamian) [reply applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: How to add data automatically?

**Pull request closed without merging**